pkg/autodiscovery/listeners/kube_endpoints.go

```diff
@@ -27,6 +27,7 @@ import (
 
 const (
 	kubeEndpointsAnnotationFormat = "ad.datadoghq.com/endpoints.instances"
+	leaderAnnotation              = "control-plane.alpha.kubernetes.io/leader"
```

Comment:
Seems like [it's not meant to change](https://github.com/kubernetes/client-go/blob/master/tools/leaderelection/resourcelock/interface.go#L30) too, I was a bit worried about the `alpha` in there.